sources/Renderer/Direct3D12/D3D12SwapChain.cpp

```diff
@@ -441,13 +448,17 @@ HRESULT D3D12SwapChain::CreateResolutionDependentResources(const Extent2D& resol
             swapChainDesc.Scaling               = DXGI_SCALING_NONE;
             swapChainDesc.SwapEffect            = DXGI_SWAP_EFFECT_FLIP_DISCARD;
             swapChainDesc.AlphaMode             = DXGI_ALPHA_MODE_IGNORE;
-            swapChainDesc.Flags                 = 0;
+            swapChainDesc.Flags                 = (tearingSupported_ ? DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING : 0u);
         }
         auto swapChain = renderSystem_.CreateDXSwapChain(swapChainDesc, wndHandle.window);
 
         swapChain.As(&swapChainDXGI_);
     }
 
+    BOOL fullscreenState;
```

Comment:
Unless it's a larger buffer that is intended to be uninitialized at allocation time (like `LLGL::DynamicByteArray{ N, LLGL::UninitializeTag{} }` for instance), I'd prefer to have all variables default initialized. Skipping the `= FALSE` here is really just a micro optimization.

Reply:
Ah yeah missed that, my bad